<launch>

	<arg name="exploration" pkg="topological_exploration" type="topological_exploration">
		<param name="sweep_type" value="complete"/>
		<param name="collection_name" value="FremenGrid"/>
		<param name="schedule_directory" value="/localhome/strands/schedules"/>
		<param name="verbose" value="false"/>
		<param name="resolution" value="0.1"/>
		<param name="dimX" value="100"/>
		<param name="dimY" value="100"/>
		<param name="dimZ" value="50"/>
		<param name="grids_directory" value="/localhome/strands/3dmaps"/>
	</node>

</launch>
